config/wallet.go

```diff
@@ -94,6 +107,11 @@ func (w *Wallet) Load(name string, passphrase string) (wallet.Account, error) {
 		return nil, err
 	}
 
+	// Run any account migration and save the config.
```

Comment:
Why would this be needed here if global migrations ran during config init?

Reply:
Or at least do not save in this case.

Reply:
Just a leftover, sorry.